pattern_matching.md

```diff
@@ -18,3 +18,97 @@ Note that some CL implementations include regexp facilities, notably
 [CLISP](http://clisp.sourceforge.net/impnotes.html#regexp) and
 [ALLEGRO CL](https://franz.com/support/documentation/current/doc/regexp.htm). If
 in doubt, check your manual or ask your vendor.
+
+Description provided below is far from complete, so don't forget to
+check the reference manual that comes along with the CL-PPCRE library.
+
+
+## Using PPCRE
+
+CL-PPCRE (abbreviation for Portable Perl-compatible regular
+expressions) is a portable regular expression library for Common Lisp
+with a broad set of features and good performance. It has been ported
+to a number of Common Lisp implementations and can be easily installed
+(or added as a dependency) via Quicklisp:
+
+~~~lisp
+(ql:quickload "cl-ppcre")
+~~~
+
+It might be more convenient to use CL-PPCRE with the CL-INTERPOL
```

Comment:
I would pop out CL-INTERPOL and put it down at the end of the section, it's a separable library and piece of information (although it can be super useful).

Reply:
I would enjoy to see a reference to cl21 (https://lispcookbook.github.io/cl-cookbook/cl21.html) too.
(Thanks for the PR !)